modules/ROOT/pages/administration/access-control/load-privileges.adoc

```diff
@@ -0,0 +1,82 @@
+:description: How to use Cypher to manage load privileges.
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE ROLE roleLoadAllData
+----
+////
+
+[role=enterprise-edition aura-db-enterprise]
+[[access-control-load-privileges]]
+= Load privileges
+
+[abstract]
+--
+This section explains how to use Cypher to manage load privileges.
+--
+
```

Comment:
We don't use abstracts any more (I have been meaning to get to deleting all existing ones, but have not got around to it yet :/) 
```suggestion
_This feature was released in Neo4j 5.13._

This section explains how to use Cypher to manage load privileges.
```